### DISCUSSION TITLE
SSO

### DISCUSSION THREAD

**Discussion by alexjpanagis** [forem/forem#17158] [Feature Requests]
Ability to use your own authentication system (i.e for a software company that wants to run a community using Forem to be able to provide a more seamless experience).

**Comment by Garrett842** [comment]
Yeah, you are right. Every forum or startup need to use it own authentication system. While implementing this ideas I have recently hired a phython expert so that he could integrate the authentication system in our [Instagram tool](https://chrome.google.com/webstore/detail/ig-scraper-email-finder-l/kjfmcpdbdabkpiekonlcajhabldpacjo) which is mostly used for finding emails of instagram account. And the data need to be secured.
